internal/commands/cnab.go

```diff
@@ -161,3 +184,51 @@ func resolveBundle(dockerCli command.Cli, name string, pullRef bool, insecureReg
 	}
 	return nil, fmt.Errorf("could not resolve bundle %q", name)
 }
+
+func requiredClaimBindMount(c claim.Claim, targetContextName string, dockerCli command.Cli) (bindMount, error) {
+	var specifiedOrchestrator string
+	if rawOrchestrator, ok := c.Parameters["docker.orchestrator"]; ok {
+		specifiedOrchestrator = rawOrchestrator.(string)
+	}
+
+	return requiredBindMount(targetContextName, specifiedOrchestrator, dockerCli)
+}
+
+func requiredBindMount(targetContextName string, targetOrchestrator string, dockerCli command.Cli) (bindMount, error) {
+	if targetOrchestrator == "kubernetes" {
+		return bindMount{false, ""}, nil
+	}
+
+	socketPath := "/var/run/docker.sock"
+	// TODO:smarter handling of default context required
+	if targetContextName == "" {
+		return bindMount{true, socketPath}, nil
+	}
+
+	ctxMeta, err := dockerCli.ContextStore().GetContextMetadata(targetContextName)
+	if err != nil {
+		return bindMount{false, ""}, err
```

Comment:
It's totally "nit", but I guess you can just return directly `return bindMount{}, err` for 2 reasons:
- the first one is that the default values when creating a bindMount are already `false` and `""`
- the second is that as you return an error, no-one should ever check the bindMount you return.

Reply:
makes sense. I've made the alterations as suggested and factored out the socket path resolving to make that easier to test